droopler.install

```diff
@@ -877,7 +871,7 @@ function droopler_update_8105() {
     'd_p_counters' => [
       'core.entity_form_display.paragraph.d_p_group_of_counters.default' => '5463576b8024a2794ad29a12a893e138',
       'core.entity_form_display.paragraph.d_p_single_counter.default' => 'e857d2f5142d2b0b29fc3c49f071eae6',
-      'core.entity_view_display.paragraph.d_p_group_of_counters.default' => 'b112cb894119aa5637c5021448914b2d',
+      'core.entity_view_display.paragraph.d_p_group_of_counters.default' => 'cc73c821346cb9222ed66f554a52246b',
```

Comment:
Ten hash nie powinien się zmienić, to na pewno jest ok?
To jeden z hashy liczonych po przejściu przez migrację Media, a nie w wersji 1.4.